### PR TITLE
chore(manager): 🌐 add translations for DO messages

### PR DESCRIPTION
**Pull request by jyyi1**
Related PR: #1118 
Internal CL: 463336189

### CODE REVIEW

File: src/server_manager/messages/sv.json [hunk 1]
```diff
@@ -240,8 +242,8 @@
   "setup-cancel": "Avbryt när du vill",
   "setup-create": "Skapa server",
   "setup-description": "Har du ingen server? Skapa ett konto med DigitalOcean.",
-  "setup-do-cost": "Endast 50 kronor i månaden",
```
Comment (fortuna) [inline src/server_manager/messages/sv.json:243]: This is wrong. It says 50 Swedish kronor. We can't convert that.

**Review by fortuna — APPROVED** [review]
Interesting to see the currency representation in different languages.
The Swedish one is broken though. We should not convert currencies. We should fix that as a follow up. Perhaps we need to reach out our localization point of contact.

File: src/server_manager/messages/sv.json (hunk 1 above)
Comment (jyyi1) [reply src/server_manager/messages/sv.json:243]: This is the old translation, it's updated to 60 SEK, is that correct?

**Review by daniellacosse — APPROVED** [review]

**Comment by jyyi1** [comment]
> Interesting to see the currency representation in different languages. The Swedish one is broken though. We should not convert currencies. We should fix that as a follow up. Perhaps we need to reach out our localization point of contact.

We can put a comment in the [`description`s](https://github.com/Jigsaw-Code/outline-server/blob/master/src/server_manager/messages/master_messages.json#L1149) to let the translation team be aware of that.

**Pull request merged**